pkg/steps/clusterinstall/clusterinstall.go

```diff
@@ -94,7 +94,7 @@ func E2ETestStep(
 		params = api.NewOverrideParameters(params, overrides)
 	}
 
-	step := steps.TemplateExecutionStep(template, params, podClient, templateClient, artifactDir, jobSpec)
+	step := steps.TemplateExecutionStep(template, params, podClient, templateClient, artifactDir, jobSpec, nil)
```

Comment:
Don't we have `config.Resources` in this case?

Reply:
It seems that we don't. But that smells, we should have. I will check what it takes to implement pass the resources there.

Reply:
addressed